content/settings/settings.rst

```diff
@@ -78,12 +78,9 @@ Administrators <../admin/admin.html>`__.
        1. The adapter used to send outgoing SMSes. Note that SMS replies
           may be sent out via a different adapter, depending on the
           adapter by which they arrived.
-       2. Our current provider choices are IntelliSMS and Twilio (refer
+       2. Our current provider is Twilio (refer
```

Comment:
Is the word 'section' on the next line supposed to link to something? Can you take a quick look?

Reply:
Yes it is supposed to link to information for administrator section which will become in the new outline "Connecting FrontlineSMS to NEMO". I will add a note about it in our review doc so it can be integrated in the changes while doing the detailed paragraph by paragraph review.